lib/stat_tracker.rb

```diff
@@ -113,4 +113,47 @@ def worst_coach(season)
 
     coach_results.min_by { |k, v| v }[0]
   end
+
+  def calculate_team_accuracies(season)
+    team_accuracies = {}
```

Comment:
to avoid the logic on line 134 ` = Hash.new { |hash, key| hash[key] = [] }`

Reply:
done!